src/Background/stat_estimators.jl

```diff
@@ -1,5 +1,44 @@
 using Statistics
 
+"""
+    sigma_clip(data, sigma_low, sigma_high; center=median, std=std, iterations=5)
+    sigma_clip(data, sigma; center=median, std=std, iteration=5)
+
+This function returns sigma clipped values of the input `data`. The funtion `sigma_clip!` is an inplace implementation and mutates the `data`.
+`sigma_high` and `sigma_low` are for un-symmetrical clipping, when `sigma_low = sigma_high` then they can be passed as `sigma`.
+`center`, `std` and `iterations` are optional parameters which are functions for finding central element, standard deviation and number of iterations of sigma
+clipping to be carried out.
+
+# Example
+```jldoctest
+julia> data = [1, 2, 3];
+
+julia> sigma_clip(data, 1)
+3-element Array{Float64,1}:
+ 1.0
+ 2.0
+ 3.0
+
+julia> sigma_clip(data, 1, 1)
+3-element Array{Float64,1}:
+ 1.0
+ 2.0
+ 3.0
+```
+"""
+
+function sigma_clip!(data::AbstractArray, sigma_low::Real, sigma_high::Real; iterations::Integer=5, center=median, std=std)
```

Comment:
I think you can define
```suggestion
function sigma_clip!(data::AbstractArray, sigma_low::Real, sigma_high::Real=sigma_low; iterations::Integer=5, center=median, std=std)
```
and remove the `sigma_clip!` method below

Reply:
ahhh that's a good way of doing it. I forgot you can self-reference the function arguments as defaults. 

Also, don't feel like you need to parametrize if you don't have too. Both for readability and generality.

Reply:
> Also, don't feel like you need to parametrize if you don't have too. Both for readability and generality.

Yes, I was also thinking about that.  I think that `clamp` doesn't have restrictions.